Pipfile

```diff
@@ -27,6 +27,7 @@ pika = "==0.10.0"
 cryptography = "==1.9"
 flask-httpauth = "==3.2.3"
 six = "==1.10.0"
+xlsxwriter = "*"
```

Comment:
We should probably pin this to a specific version

Reply:
It should be 1.0.2